Signal/src/ViewControllers/OWS2FARegistrationViewController.m

```diff
@@ -60,171 +56,53 @@ - (void)viewDidLoad
 
     self.view.backgroundColor = UIColor.whiteColor;
 
-    [self createContents];
+    PinEntryView *entryView = [PinEntryView new];
+    self.entryView = entryView;
+    entryView.delegate = self;
+    [self.view addSubview:entryView];
+    
+    entryView.instructionsText = NSLocalizedString(@"REGISTER_2FA_INSTRUCTIONS", @"Instructions to enter the 'two-factor auth pin' in the 2FA registration view.");
+    
+    // Layout
+    
+    [entryView autoPinEdgesToSuperviewMargins];
```

Comment:
Is this safe for iPhone X?  I thought we had to use `autoPinTopToSuperviewWithMargin` for top and bottom for this kind of root view.

Reply:
I'd like to address this in the pursuit of: "Make 2FA screen look more like the rest of the registration flow." since the layout here's going to get changed regardless.